xs2a-adapter-service-impl/src/test/java/de/adorsys/xs2a/adapter/impl/http/wiremock/WiremockStubDifferenceDetectingInterceptorTest.java

```diff
@@ -58,6 +58,24 @@ void postHandle_allMatch() throws JsonProcessingException {
             .doesNotContainKey(ResponseHeaders.X_GTW_ASPSP_CHANGES_DETECTED);
     }
 
+    @Test
+    void postHandle_ContentTypeNotMatching() throws JsonProcessingException {
+        RequestBuilderImpl request = new RequestBuilderImpl(httpClient, "POST", "https://bank.com/v1/consents");
```

Comment:
There is a constant for POST consent url - CONSENTS_URL

Reply:
fixed, thanks